motoman_sda10f_support/config/sda10f_motion_interface.yaml

```diff
@@ -1,4 +1,4 @@
-topic_list
+topic_list:
```

Comment:
Took me some time to spot this :), but I think you'll want the colon back, right?